src/gui/optionsdialog.ui

```diff
@@ -1395,17 +1395,44 @@ readme[0-9].txt: filter 'readme1.txt', 'readme2.txt' but not 'readme10.txt'.</st
             <item>
              <widget class="QGroupBox" name="autoRunBox">
               <property name="title">
-               <string>Run e&amp;xternal program on torrent completion</string>
-              </property>
-              <property name="checkable">
-               <bool>true</bool>
-              </property>
-              <property name="checked">
-               <bool>false</bool>
+               <string>Run external program</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_19">
                <item>
-                <widget class="QLineEdit" name="lineEditAutoRun"/>
+                <widget class="QGroupBox" name="groupBoxRunOnAdded">
+                 <property name="title">
+                  <string>on torrent added</string>
```

Comment:
```suggestion
                  <string>On torrent added</string>
```

Reply:
The intention was, as it were, to continue the general preamble, i.e. `Run external program` + `on torrent added`. On torrent added doesn't look good as separate statement. Maybe `Run on torrent added`?

Reply:
>Maybe `Run on torrent added` (i.e. repeat Run in each suboption)?

Remade it this way for now.